servicebus/spring-cloud-azure-starter-integration-servicebus/single-namespace/src/main/resources/application.yaml

```diff
@@ -2,4 +2,4 @@ spring:
   cloud:
     azure:
       servicebus:
-        connection-string: ${AZURE_SERVICEBUS_CONNECTION_STRING}
+        namespace: sb-servicebusnamespace
```

Comment:
I don't think you can hardcode this value, it should be created by terraform on the fly